draft-ietf-masque-h3-datagram.md

```diff
@@ -139,20 +145,27 @@ an error is not mandatory in this case because HTTP/3 implementations might have
 practical barriers to determining the active stream concurrency limit that is
 applied by the QUIC layer.
 
-HTTP/3 datagrams MUST only be sent with an association to a stream that supports
-semantics for HTTP Datagrams. For example, existing HTTP methods GET and POST do
-not define semantics for associated HTTP Datagrams; therefore, HTTP/3 datagrams
-cannot be sent associated with GET or POST request streams. If an endpoint
-receives an HTTP/3 datagram associated with a method that has no known semantics
-for HTTP Datagrams, it MUST abort the corresponding stream with
-H3_DATAGRAM_ERROR. Future extensions MAY remove these requirements if they
-define semantics for such HTTP Datagrams and negotiate mutual support.
+HTTP/3 datagrams MUST only be sent with an association to a stream whose HTTP
+semantics explicitly supports HTTP Datagrams. For example, existing HTTP
```

Comment:
This is still a bit imprecise. suggest

```suggestion
HTTP/3 datagrams MUST only be sent with an association to a stream whose HTTP
semantics explicitly supports HTTP Datagrams (usually, in conjunction with the extension that invokes
the use of HTTP Datagrams. For example, existing HTTP
```

Also, these requirements seem general to all HTTP datagrams, but are placed against "HTTP/3 datagrams"